packages/alice/lib/ui/calls_list/widget/alice_log_list_widget.dart

```diff
@@ -32,9 +32,10 @@ class _AliceLogListWidgetState extends State<AliceLogListWidget> {
 
   @override
   Widget build(BuildContext context) {
-    return ValueListenableBuilder<List<AliceLog>>(
-      valueListenable: widget.logsListenable,
-      builder: (_, List<AliceLog> logs, __) {
+    return StreamBuilder<List<AliceLog>>(
+      stream: widget.logsStream,
+      builder: (context, snapshot) {
```

Comment:
What if the snapshot is still loading? What if the snapshot has an error?

Reply:
Fixed.